db/migrate/20160404152107_add_subscription_id_to_ext_management_system.rb

```diff
@@ -0,0 +1,9 @@
+class AddSubscriptionIdToExtManagementSystem < ActiveRecord::Migration[5.0]
+  def up
+    add_column :ext_management_systems, :subscription_id, :string
```

Comment:
`_id` should always be Rails foreign keys (in our case bigint).  I think the column name should be something else.